src/Hocon.Configuration.Test/SerializationSpecs.cs

```diff
@@ -42,8 +42,25 @@ public void ShouldSerializeHocon(string hocon, string fallback1, string fallback
         private void VerifySerialization(Config config)
         {
             var serialized = JsonConvert.SerializeObject(config);
-            var deserialized = (Config)JsonConvert.DeserializeObject(serialized);
+            var deserialized = JsonConvert.DeserializeObject<Config>(serialized);
```

Comment:
LOL - was this really as simple as just using the generic method?

Reply:
yup... its just as simple as that